demo/api/schema.gql

```diff
@@ -1133,21 +1142,27 @@ input ProductInput {
   discounts: [ProductDiscountsInput!]! = []
   articleNumbers: [String!]! = []
   dimensions: ProductDimensionsInput
-  statistics: ProductStatisticsInput
-  variants: [ProductVariantInput!]! = []
+  statistics: ProductProductStatisticsInput
+  variants: [ProductProductVariantInput!]! = []
```

Comment:
We need this "Product" prefix because the product variant API is generated in the same folder, right? Could we make this ProductNestedProductVariantInput or NestedProductVariantInput instead?

Reply:
yes.

good idea, changed